src/software/gui/geometry_conversion.h

```diff
@@ -22,6 +22,7 @@ QPointF createQPointF(const Point& point);
  * @return The QRectF representation of the given Rectangle
  */
 QRectF createQRectF(const Rectangle& rectangle);
+Rectangle createRectangle(const QRectF& rectangle);
```

Comment:
please update the jdoc

Reply:
Whoops I actually ended up not using that. Removed it.